### PR TITLE
chore: additional values for pr deploy

### PR DESCRIPTION
**Pull request by rokroskar**
allows specifying additional values to be passed to a custom deployment via the `extra-values` option:

/deploy renku-ui=0.11.x extra-values=ui.canary.enabled=true,ui.canary.image.repository=renku/renku-ui,ui.canary.image.tag=0.11.11-n044.hbfc2e2b

### CODE REVIEW

File: actions/check-pr-description/action.yaml [hunk 1]
```diff
@@ -29,6 +29,9 @@ outputs:
   renku-ui:
     description: renku-ui reference as specified in the command string
     value: ${{ steps.check-string.outputs.renku-ui }}
+  extra-values:
+    description: extra values passed to helm
```
[the remark this answers is not in the document]
Comment (ableuler) [reply actions/check-pr-description/action.yaml:33]: I guess one should only use this flag once even when providing multiple values. Or will it work the other way too?

**Review by ableuler — CHANGES_REQUESTED** [review]
Very useful feature!

File: actions/check-pr-description/action.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (rokroskar) [reply actions/check-pr-description/action.yaml:33]: yes, only once - this was the simplest way I could think of implementing it

[the remark this answers is not in the document]
Comment (ableuler) [reply actions/check-pr-description/action.yaml:33]: 👍 that's perfectly fine, just tried to make this a bit more obvious in the commit suggestion above.

**Review by ableuler — APPROVED** [review]

**Comment by rokroskar** [comment]
hmm something broke in the deployment... need to debug

**Review by ableuler — APPROVED** [review]

**Pull request merged**